### PR TITLE
Test: create operations e2e test

### PR DESCRIPTION
**Pull request by BCerki**
Part of #139. See readme for test-running instructions

I don't think the sonarcloud fail is a problem since the test endpoint will never exist anywhere but dev, but if reviewers disagree just let me know!

### CODE REVIEW

**Comment by shon-button** [comment]
(WIP)
image

File: bc_obps/registration/api.py [hunk 1]
```diff
@@ -19,10 +19,32 @@
...
+from django.http import HttpResponse
+import os
 
 router = Router()
 
+# testing endpoint
+@router.get("/test-setup")
+def setup(request):
+    print(settings.ENVIRONMENT)
+    print(type(settings.ENVIRONMENT))
```
Comment (andrea-williams) [inline bc_obps/registration/api.py:37]: do we still need these print statements? It's not a big deal now, but we'll want to make sure they're removed before we go to production

File: client/e2e/operations.spec.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import { test, expect } from "@playwright/test";
+
+// Annotate entire file as serial.
+test.describe.configure({ mode: "serial" });
+
+test.beforeEach(async ({ page }, testInfo) => {
+  // eslint-disable-next-line no-console
+  console.log(`Running ${testInfo.title}`);
+  // reset db and confirm there are no errors setting up the test data
+  await page.goto("http://localhost:8000/api/registration/test-setup");
```
Comment (andrea-williams) [inline client/e2e/operations.spec.ts:10]: I think we'll need to parameterize these URLs in order for them to work in Openshift environments...?

Comment (BCerki) [reply client/e2e/operations.spec.ts:10]: ... maybe? Do we normally run e2e tests in Openshift, or just locally/in github actions?

Comment (andrea-williams) [reply client/e2e/operations.spec.ts:10]: fair point. I don't know if we do on CAS (haven't been paying attention). I assumed so but that's entirely baseless. Any of the devopsy people (e.g., @dleard  , @Sepehr-Sobhani  or @mikevespi) able to shed light on this?

**Review by mikevespi — APPROVED** [review]
👍

File: client/e2e/operations.spec.ts (hunk 2 above)
Comment (dleard) [reply client/e2e/operations.spec.ts:10]: only in gh actions

**Pull request merged**